parser-typechecker/tests/Unison/Test/TermPrinter.hs

```diff
@@ -151,7 +151,7 @@ test = scope "termprinter" . tests $
   , tc "case e of { a } -> z"
   , pending $ tc "case e of { () -> k } -> z" -- TODO doesn't parse since 'many leaf' expected before the "-> k"
                                               -- need an actual effect constructor to test this with
-  , tc "if a then (if b then c else d) else e"
+  , tc "if a then if b then c else d else e"

```

Comment:
Are you sure the new way is better, in this example?

Reply:
In this example as a one liner, no. 🥴 If you break it onto multiple lines (which will often happen for more realistic examples) it seems ok to me.